app/domain/rotation/master_rotator.rb

```diff
@@ -28,7 +31,17 @@ def rotate_every(seconds)
     end
 
     def rotate_all
-      scheduled_rotations.each(&:run)
+      # Attempt an advisory lock to prevent other servers from trying to rotate
+      # at the same time or immediately after another server rotated the secret.
+      # The same `id` needs to be used across all servers talking to the same
+      # database, but can be any unused integer.
+      id = 8675309
```

Comment:
This needs to be something static and definitely not within the rotate function itself. Can we take a hash of something stable to generate it?

Reply:
Updated to use a string specified under initializers for the rotator that gets converted to integer at point in time (used the string "possum" for now).